File: app/src/main/java/com/example/hellooboe/ActivityLoadFile.java
package com.example.hellooboe;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

public class ActivityLoadFile extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_load);
        setUpRecyclerView();
    }

    private void setUpRecyclerView() {
        RecyclerView recyclerViewLoadFile = (RecyclerView) findViewById(R.id.recycler_view_load_file);
        RecyclerView.LayoutManager linearLayoutManagerLoadFile = new LinearLayoutManager(this);
        recyclerViewLoadFile.setLayoutManager(linearLayoutManagerLoadFile);
        //TODO Load strings from c++ SaveFiles
        String[] saveFileNames = {"a", "b", "c", "d", "e"};
        RecyclerView.Adapter recyclerViewAdapterLoadFile = new AdapterLoadFile(saveFileNames);
        recyclerViewLoadFile.setAdapter(recyclerViewAdapterLoadFile);
    }

}

File: app/src/main/java/io/fourth_finger/sound_sculptor/ActivityLoadFile.kt
package io.fourth_finger.sound_sculptor

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class ActivityLoadFile : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_load)
        setUpRecyclerView()
    }

    private fun setUpRecyclerView() {
        val recyclerViewLoadFile = findViewById<View>(R.id.recycler_view_load_file) as RecyclerView
        val linearLayoutManagerLoadFile: RecyclerView.LayoutManager = LinearLayoutManager(this)
        recyclerViewLoadFile.layoutManager = linearLayoutManagerLoadFile
        //TODO Load strings from c++ SaveFiles
        val saveFileNames = arrayOf("a", "b", "c", "d", "e")
        val recyclerViewAdapterLoadFile: RecyclerView.Adapter<*> = AdapterLoadFile(saveFileNames)
        recyclerViewLoadFile.adapter = recyclerViewAdapterLoadFile
    }
}